example/scalalib/native/1-simple/build.mill

```diff
@@ -0,0 +1,29 @@
+package build
+import mill._, scalalib._, scalanativelib._
+
+object `package` extends RootModule with ScalaNativeModule {
+  def scalaVersion = "3.3.4"
+  def scalaNativeVersion = "0.5.5"
+
+  // You can have arbitrary numbers of third-party dependencies
+  def ivyDeps = Agg(
+    ivy"com.lihaoyi::mainargs::0.7.6"
+  )
+
+  object test extends ScalaNativeTests with TestModule.Utest{
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+}
+
+// This example demonstrates a simple Scala program that generates HTML content
+// from user-provided text and prints it to the standard output, utilizing Scala
+// Native for native integration and `mainargs` for command-line argument parsing.
+
+/** Usage
+
+> ./mill show nativeLink  # Build and link native binary
+...
+done compiling
+
```

Comment:
Let's include the `"../out/nativeLink.dest/blah"` executable path so people can see it, and also run the native executable in a subsequent command to show the output